test/parallel/test-crypto-sign-verify.js

```diff
@@ -369,3 +369,33 @@ common.expectsError(
     () => crypto.createSign('sha8'),
     /Unknown message digest/);
 }
+
+[
+  { private: fixtures.readSync('test_ed25519_privkey.pem', 'ascii'),
+    public: fixtures.readSync('test_ed25519_pubkey.pem', 'ascii'),
+    sigLen: 64 },
+  { private: fixtures.readSync('test_ed448_privkey.pem', 'ascii'),
+    public: fixtures.readSync('test_ed448_pubkey.pem', 'ascii'),
+    sigLen: 114 }
+].forEach((pair) => {
+  const data = Buffer.from('Hello world');
+
+  {
+    const sig = crypto.signOneShot(data, pair.private);
+    assert.strictEqual(sig.length, pair.sigLen);
+
+    assert.strictEqual(crypto.verifyOneShot(data, pair.private, sig), true);
+    assert.strictEqual(crypto.verifyOneShot(data, pair.public, sig), true);
```

Comment:
There should also be tests for verification failures (e.g. using the wrong key, the wrong message, the wrong signature).

Reply:
Added.